naming/src/main/java/com/alibaba/nacos/naming/healthcheck/MysqlHealthCheckProcessor.java

```diff
@@ -173,7 +173,7 @@ public void run() {
                 Throwable cause = t;
                 int maxStackDepth = 50;
                 for (int deepth = 0; deepth < maxStackDepth && cause != null; deepth++) {
-                    if (cause instanceof SocketTimeoutException || cause instanceof ConnectTimeoutException
+                    if (cause instanceof SocketTimeoutException || cause instanceof ConnectException
```

Comment:
io.netty.channel.ConnectTimeoutException extends java.net.ConnectException; So replace it to be compatible.